processing/src/main/java/org/apache/druid/segment/nested/NestedDataColumnSerializer.java

```diff
@@ -551,12 +552,12 @@ int lookupGlobalId(Long value)
     }
 
     @Override
-    void openColumnSerializer(String field, SegmentWriteOutMedium medium, int maxId) throws IOException
+    public void open(String field) throws IOException
```

Comment:
Just from looking at the diff, I'm a bit lost as to why all of these parameters were able to be eliminated...  I think it's because I've lost track of which class I'm looking at.

Reply:
ah the diff is making this look funny, i changed long and double columns to override `open` instead of `openColumnSerializer` (open happens before the first pass and opens the intermediary dictionaryid int column that has the unsorted local ids) to also open the long and double column serializers at this phase.

`openColumnSerializer` happens in `writeTo` when writing out the compressed dictionary id int column with the sorted localIds.

I think `openColumnSerializer` could just be moved into `writeTo` since nothing is overriding it